timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -265,8 +285,7 @@ export default {
       showLeftPanel: true,
       leftPanelTab: 0,
       leftPanelTabItems: ['Explore', 'Investigate'],
-      renameScenarioDialog: false,
```

Comment:
not sure if that was by intention to remove the renameScenarioDialog here. The PR seems fine for me, but I let you merge it to confirm it was not a misstake.